xml/sle_update_online.xml

```diff
@@ -367,21 +367,21 @@ Available migrations:
         Desktop Applications Module 15 SP2 x86_64
         Python 2 Module 15 SP2 x86_64
         Server Applications Module 15 SP2 x86_64
-       
+
     2 | SUSE Linux Enterprise Server 15 SP1 x86_64
         Basesystem Module 15 SP1 x86_64
         Desktop Applications Module 15 SP1 x86_64
         Python 2 Module 15 SP1 x86_64
-        Server Applications Module 15 SP1 x86_64       
+        Server Applications Module 15 SP1 x86_64
 
 [num/q]: </screen>
 
 
-    
-    
-    
-    
-    
+
+
+
+
+
```

Comment:
Can you separate the whitespace changes from the actual bugfix?

Reply:
next time, sorry.

Reply:
You are making whitespace changes with every commit. Seems automatic reformatting is not necessarily desired. Anyway, not a blocker for this PR.